### PR TITLE
Register Archive Posts and Search Widgets translatable elements

### PR DESCRIPTION
**Pull request by amElnagdy**

### CODE REVIEW

File: src/class-wpml-elementor-translatable-nodes.php [hunk 1]
```diff
@@ -719,6 +719,56 @@ public static function get_nodes_to_translate() {
 					),
 				),
 			),
+			 'archive-posts' => array(
+                'conditions' => array( self::TYPE => 'archive-posts' ),
```
Comment (strategio) [inline src/class-wpml-elementor-translatable-nodes.php]: @amElnagdy, could you please check your indentation?

Comment (amElnagdy) [reply src/class-wpml-elementor-translatable-nodes.php]: sorry, should be ok now.

File: src/class-wpml-elementor-translatable-nodes.php [hunk 2]
```diff
@@ -719,6 +719,56 @@ public static function get_nodes_to_translate() {
...
+            'search-form' => array(
+                'conditions' => array( self::TYPE => 'search-form' ),
+                'fields'     => array(
+                    array(
+                        'field'       => 'placeholder',
+                        'type'        => __( 'Placeholder', 'sitepress' ),
+                        'editor_type' => 'LINE'
+                    ),
+                ),
+            ),
```
Comment (strategio) [inline src/class-wpml-elementor-translatable-nodes.php]: @amElnagdy, there's still something wrong with the indentation. Do you have the same settings as below?

indentation

Comment (amElnagdy) [reply src/class-wpml-elementor-translatable-nodes.php]: not really, the Use tab character and smart tabs options were not selected.

**Comment by strategio** [comment]
Looks good now @amElnagdy +1

**Review by strategio — DISMISSED** [review]

**Review by andreasciamanna — DISMISSED** [review]

**Pull request merged**